crates/matrix-sdk-crypto/src/verification/sas/sas_state.rs

```diff
@@ -239,21 +240,22 @@ impl<S: Clone + std::fmt::Debug> std::fmt::Debug for SasState<S> {
 /// The initial SAS state.
 #[derive(Clone, Debug)]
 pub struct Created {
-    protocol_definitions: SasV1Content,
+    pub protocol_definitions: SasV1Content,
```

Comment:
isn't the compiler/clippy complaining that you made this public but don't give doc comments for it (also the ones below)? Not that I think it makes a lot of sense here, it is pretty clear naming ... just curious...

Reply:
The struct and field are public in the module and in the parent module, but it's never exposed further from the parent module.